assembl/static2/js/app/components/search.jsx

```diff
@@ -91,19 +91,33 @@ if (__resourceQuery) {
   };
   // }
 } else {
-  Link = require('react-router').Link; // eslint-disable-line
+  // Link = require('react-router').Link; // eslint-disable-line
+  Link = (props) => {
+    return <a href={props.to} dangerouslySetInnerHTML={props.dangerouslySetInnerHTML} />;
+  };
+  const slug = document.getElementById('discussion-slug').value;
   getUrl = (hit) => {
     const id = hit._source.id;
+    let ideaBase64id;
+    let postBase64id;
+    let ideaId;
     switch (hit._type) {
     case 'synthesis':
-      return `posts/${id}`;
+      return undefined;
     case 'user':
-      return `profile/${id}`;
```

Comment:
those 2 cases were copy/paste from v1 ?

Reply:
yep, this was when we didn't even had v2 routes